ITE/4/README.adoc

```diff
@@ -430,35 +432,76 @@ resulting build and CI report.
 ==== Use Case 3: Granular JSON Access
 
 The enhancement proposed in this ITE can also be used to provide more granular
-access to certain resources, such as JSON files. A generic URI can be used to
-resolve to the information contained in a specific key of a JSON file. This allows
-for greater flexibility when using in-toto with different types of artifacts.
+access to resources such as JSON files. A generic URI can be used to resolve to
+the information contained in a specific key of a JSON file using JSONPath. This
```

Comment:
Could you explain how to escape JSONPath w/o conflicting with `fnmatch` metacharacters?

Reply:
Resolving in favour of discussion below.